bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Epoch.scala

```diff
@@ -1,10 +1,17 @@
 package edu.gemini.spModel.core
 
-/** Epoch in Julian years. */
-final case class Epoch(value: Double)
+import scalaz._, Scalaz._
+
+/** Epoch in Gregorian (?) years. */
+final case class Epoch(year: Double)
```

Comment:
This is the epoch of observation for targets with proper motion, which is not always 2000.0.